tools/lotus2/lotus2.xml

```diff
@@ -171,7 +195,8 @@ exit \$EXIT_VALUE
         </conditional>
         <param argument="-map" type="data" format="tabular" optional="true" label="Mapping file (optional)" help="Needed to demultiplex the FASTQ files using sdm. If the FASTQ are already demultiplexed, this can be omitted." />
         <param argument="-platform" type="select" label="Sequencing platform">
-            <option value="miSeq" selected="true">miSeq</option>
+            <option value="" style="color:gray" selected="true">(Default)</option>
```

Comment:
```suggestion
            <option value="" selected="true">(Default)</option>
```

Reply:
There's a difference between not selecting anything, and selecting miSeq (the difference being that in the first case lotus attempts to figure it out on its own)

Reply:
I understand, I am just suggesting to remove the `style` here?